style.css

```diff
@@ -1,37 +1,33 @@
-body {
+html {
   background-color: rgb(19, 19, 143);
-  font-size: 20px;
-  font-family: sans-serif, Arial, Helvetica;
-  margin: 0;
-  padding: 0;
+  color: #fff;
+  font-family: Arial, Helvetica, sans-serif;
 }
 
-.main {
-  color: #fff;
-  background-color: rgb(19, 19, 143);
-  line-height: 30px;
-  font-size: 29px;
+.container {
```

Comment:
- [ ] Please stick with the design provided in the template the background image is different than the one in [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)
![image](https://user-images.githubusercontent.com/74744787/158880119-444cc7d2-9cd4-452f-ae89-96de928c7f4f.png)
![image](https://user-images.githubusercontent.com/74744787/158880234-2c605b9a-2f83-4b5a-a7d0-55b3cb7eadcc.png)

Reply:
resolved 👍